cpp/include/cudf/io/detail/orc.hpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2019i-2020, NVIDIA CORPORATION.
```

Comment:
If this is a new file then it does not need 2019. Also there is an extra 'i' in there.

Reply:
But these functions were copied from another file as is, so wasn't sure about that.

Reply:
If the file was just renamed keeping the old date would be ok but this is really a new file. The old file will still have its own history.

Reply:
I have made the changes.